com.chartboost.helium/Runtime/Platforms/HeliumExternal.cs

```diff
@@ -40,95 +31,118 @@ protected static bool CheckInitialized()
+            HeliumLogger.Log(LogTag, $"InitWithAppIdAndSignature {appId}, {appSignature} and version {Application.unityVersion}");
             HeliumEventProcessor.Initialize();
         }
         
         public virtual void SetSubjectToCoppa(bool isSubject)
         {
-            Log($"SetSubjectToCoppa {isSubject}");
+            HeliumLogger.Log(LogTag, $"SetSubjectToCoppa {isSubject}");
         }
         
         public virtual void SetSubjectToGDPR(bool isSubject)
         {
-            Log($"SetSubjectToGDPR {isSubject}");
+            HeliumLogger.Log(LogTag, $"SetSubjectToGDPR {isSubject}");
         }
 
         public virtual void SetUserHasGivenConsent(bool hasGivenConsent)
         {
-            Log($"SetUserHasGivenConsent {hasGivenConsent}");
+            HeliumLogger.Log(LogTag, $"SetUserHasGivenConsent {hasGivenConsent}");
         }
 
         public virtual void SetCCPAConsent(bool hasGivenConsent)
         {
-            Log($"SetCCPAConsent {hasGivenConsent}");
+            HeliumLogger.Log(LogTag, $"SetCCPAConsent {hasGivenConsent}");
         }
 
         public virtual void SetUserIdentifier(string userIdentifier)
         {
-            Log($"SetUserIdentifier {userIdentifier}");
+            HeliumLogger.Log(LogTag, $"SetUserIdentifier {userIdentifier}");
         }
 
         public virtual string GetUserIdentifier()
         {
-            Log("GetUserIdentifier");
+            HeliumLogger.Log(LogTag, "GetUserIdentifier");
             return string.Empty;
         }
 
         public virtual void Destroy()
         {
-            Log("Destroy");
+            HeliumLogger.Log(LogTag, "Destroy");
         }
 
         public virtual void Pause(bool paused)
         {
-            Log("pause");
+            HeliumLogger.Log(LogTag, "pause");
         }
 
         public virtual bool OnBackPressed()
         {
-            Log("OnBackPressed");
+            HeliumLogger.Log(LogTag, "OnBackPressed");
             return CheckInitialized();
         }
 
-        public virtual HeliumInterstitialAd GetInterstitialAd(string placementName)
+        public HeliumInterstitialAd GetInterstitialAd(string placementName)
         {
-            Log($"GetInterstitialAd at placement: {placementName}");
-            return null;
-        }
-        
-        public virtual HeliumRewardedAd GetRewardedAd(string placementName)
-        {
-            Log($"GetRewardedAd at placement: {placementName}");
-            return null;
+            HeliumLogger.Log(LogTag, $"GetInterstitialAd at placement: {placementName}");
+            if (!CanFetchAd(placementName))
+                return null;
+            try
+            {
+                return new HeliumInterstitialAd(placementName);
+            }
+            catch (Exception e)
+            {
+                HeliumLogger. LogError(LogTag, $"interstitial failed to be obtained {e}");
```

Comment:
In case it can't easily be seen
```suggestion
                HeliumLogger.LogError(LogTag, $"interstitial failed to be obtained {e}");
```